commit 68950eb8e197ba9f7ce8c9947035354e45ecc11b
Author: Susan Franklin <sfranklin>
Date:   Tue Apr 8 16:51:11 2008 +0000

    Bug 226116 - [ui] clean up unused messages

3	23	bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java
0	1	bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/UpdateAndInstallDialog.java
3	23	bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties
4	4	bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/updates/AutomaticUpdatesPopup.java
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java
index 5b8d1c8..9516426 100644
--- a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java
+++ b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/ProvSDKMessages.java
@@ -24,8 +24,6 @@ public class ProvSDKMessages extends NLS {
 		// load message values from bundle file
 		NLS.initializeMessages(BUNDLE_NAME, ProvSDKMessages.class);
 	}
-	public static String AddColocatedRepositoryDialog_InvalidURL;
-	public static String AddColocatedRepositoryDialog_MissingProfile;
 	public static String AddColocatedRepositoryDialog_Title;
 	public static String ExternalFileHandler_BundleInstalledOK;
 	public static String ExternalFileHandler_ErrorCopyingFile;
@@ -33,8 +31,6 @@ public class ProvSDKMessages extends NLS {
 	public static String ExternalFileHandler_ErrorLoadingFromZipDirectory;
 	public static String ExternalFileHandler_PromptForInstallBundle;
 	public static String ExternalFileHandler_RepositoryGeneratedOK;
-	public static String ExternalFileHandler_UnexpectedError;
-	public static String ExternalFileHandler_UserGeneratedRepoName;
 	public static String ExternalFileHandler_PromptForUnzip;
 	public static String ExternalFilesPreferencePage_AddBundleGroup;
 	public static String ExternalFilesPreferencePage_ArchiveRepoGenerationGroup;
@@ -44,11 +40,6 @@ public class ProvSDKMessages extends NLS {
 	public static String ExternalFilesPreferencePage_GenerateArchiveRepoAlways;
 	public static String ExternalFilesPreferencePage_GenerateArchiveRepoNever;
 	public static String ExternalFilesPreferencePage_GenerateArchiveRepoPrompt;
-	public static String ExternalFilesPreferencePage_GenerateRepoAlways;
-	public static String ExternalFilesPreferencePage_GenerateRepoNever;
-	public static String ExternalFilesPreferencePage_GenerateRepoPrompt;
-	public static String ExternalFilesPreferencePage_RepoGenerationGroup;
-	public static String RepositoryManipulationDialog_ErrorRetrievingSites;
 	public static String RepositoryManipulationDialog_AddButton;
 	public static String RepositoryManipulationDialog_LocationColumnHeader;
 	public static String RepositoryManipulationDialog_NameColumnHeader;
@@ -56,7 +47,6 @@ public class ProvSDKMessages extends NLS {
 	public static String RepositoryManipulationDialog_RemoveButton;
 	public static String RepositoryManipulationDialog_RemoveOperationLabel;
 	public static String RepositoryManipulationDialog_UpdateSitesDialogTitle;
-	public static String UpdateAndInstallDialog_AccessibilityFilterDescription;
 	public static String UpdateAndInstallDialog_AddSiteButtonText;
 	public static String UpdateAndInstallDialog_AddSiteOperationlabel;
 	public static String UpdateAndInstallDialog_AvailableSoftware;
@@ -69,12 +59,10 @@ public class ProvSDKMessages extends NLS {
 	public static String UpdateAndInstallDialog_RemoveSiteOperationLabel;
 	public static String UpdateAndInstallDialog_RevertActionLabel;
 	public static String UpdateAndInstallDialog_Title;
-	public static String UpdateAndInstallDialog_TypeFilterText;
 	public static String UpdateAndInstallDialog_Properties;
 	public static String UpdateAndInstallDialog_Refresh;
 
 	public static String UpdateAndInstallDialog_ViewByCategory;
-	public static String UpdateAndInstallDialog_ViewByMenuToolTip;
 	public static String UpdateAndInstallDialog_ViewByName;
 	public static String UpdateAndInstallDialog_ViewBySite;
 	public static String UpdateHandler_CannotLaunchUI;
@@ -111,7 +99,6 @@ public class ProvSDKMessages extends NLS {
 	public static String SchedulerStartup_10PM;
 	public static String SchedulerStartup_11PM;
 	public static String SchedulerStartup_12AM;
-	public static String AutomaticUpdatesJob_AutomaticUpdateSearch;
 	public static String AutomaticUpdatesPopup_PrefLinkOnly;
 	public static String AutomaticUpdatesPopup_RemindAndPrefLink;
 	public static String AutomaticUpdatesPopup_ReminderJobTitle;
@@ -120,13 +107,11 @@ public class ProvSDKMessages extends NLS {
 	public static String AutomaticUpdateScheduler_60Minutes;
 	public static String AutomaticUpdateScheduler_240Minutes;
 	public static String AutomaticUpdateScheduler_UpdateNotInitialized;
-	public static String AutomaticUpdatesDialog_DownloadedNotification;
-	public static String AutomaticUpdatesDialog_UpdatesAvailableTitle;
+	public static String AutomaticUpdatesPopup_UpdatesAvailableTitle;
 	public static String AutomaticUpdater_AutomaticDownloadOperationName;
 	public static String AutomaticUpdater_ClickToReviewUpdates;
 	public static String AutomaticUpdater_ClickToReviewUpdatesWithProblems;
 	public static String AutomaticUpdater_ErrorCheckingUpdates;
-	public static String AutomaticUpdatesDialog_UpdatesFoundNotification;
 	public static String AutomaticUpdatesPreferencePage_UpdateSchedule;
 	public static String AutomaticUpdatesPreferencePage_findOnStart;
 	public static String AutomaticUpdatesPreferencePage_findOnSchedule;
@@ -137,17 +122,12 @@ public class ProvSDKMessages extends NLS {
 	public static String AutomaticUpdatesPreferencePage_RemindGroup;
 	public static String AutomaticUpdatesPreferencePage_RemindSchedule;
 	public static String AutomaticUpdatesPreferencePage_RemindOnce;
-	public static String AutomaticUpdatesJob_Updates;
-	public static String AutomaticUpdatesDialog_ClickToReviewDownloaded;
-	public static String AutomaticUpdatesDialog_ClickToReviewNotDownloaded;
+	public static String AutomaticUpdatesPopup_ClickToReviewDownloaded;
+	public static String AutomaticUpdatesPopup_ClickToReviewNotDownloaded;
 	public static String ProvisioningPreferencePage_AlwaysOpenWizard;
 	public static String ProvisioningPreferencePage_BrowsingPrefsGroup;
 	public static String ProvisioningPreferencePage_ShowLatestVersions;
 	public static String ProvisioningPreferencePage_ShowAllVersions;
-	public static String ProvisioningPreferencePage_gcGroup;
-	public static String ProvisioningPreferencePage_enableGC;
-	public static String ProvisioningPreferencePage_gcImmediately;
-	public static String ProvisioningPreferencePage_gcRetentionTime;
 	public static String ProvisioningPreferencePage_NeverOpenWizard;
 	public static String ProvisioningPreferencePage_OpenWizardIfInvalid;
 	public static String ProvisioningPreferencePage_PromptToOpenWizard;
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/UpdateAndInstallDialog.java b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/UpdateAndInstallDialog.java
index 70f99da..ab011c5 100644
--- a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/UpdateAndInstallDialog.java
+++ b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/UpdateAndInstallDialog.java
@@ -60,7 +60,6 @@ public class UpdateAndInstallDialog extends TrayDialog implements IViewMenuProvi
 	private static final String DIALOG_SETTINGS_SECTION = "UpdateAndInstallDialog"; //$NON-NLS-1$
 	private static final String SELECTED_TAB_SETTING = "SelectedTab"; //$NON-NLS-1$
 	private static final String AVAILABLE_VIEW_TYPE = "AvailableViewType"; //$NON-NLS-1$
-	static final String INITIAL_TEXT = ProvSDKMessages.UpdateAndInstallDialog_TypeFilterText;
 
 	String profileId;
 	AvailableIUViewQueryContext queryContext;
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties
index 3da5fd8..e70192d 100644
--- a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/messages.properties
@@ -8,8 +8,6 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 ###############################################################################
-AddColocatedRepositoryDialog_InvalidURL=The URL is not valid.
-AddColocatedRepositoryDialog_MissingProfile=Could not retrieve a description of the running system.
 AddColocatedRepositoryDialog_Title=Add Site
 ExternalFileHandler_BundleInstalledOK=Bundle was successfully installed in {0}
 ExternalFileHandler_ErrorCopyingFile=Error while trying to copy the file {0}
@@ -17,8 +15,6 @@ ExternalFileHandler_ErrorExpandingArchive=Error while expanding the archive {0}
 ExternalFileHandler_ErrorLoadingFromZipDirectory=Error reading the content expanded in {0}
 ExternalFileHandler_PromptForInstallBundle="{0}" is a software plug-in.  Do you want to install it?  (Requires restart when finished).
 ExternalFileHandler_RepositoryGeneratedOK=Repository was successfully generated at {0}
-ExternalFileHandler_UnexpectedError=Unexpected error while processing {0}.
-ExternalFileHandler_UserGeneratedRepoName=User-Generated Content from {0}
 ExternalFileHandler_PromptForUnzip=Do you want to expand the contents of {0} and make it available for install?
 ExternalFilesPreferencePage_AddBundleGroup=When I add a plug-in file as an update site
 ExternalFilesPreferencePage_ArchiveRepoGenerationGroup=When I add an unrecognized archive file as an update site
@@ -28,19 +24,13 @@ ExternalFilesPreferencePage_AutoInstallBundlePrompt=Ask &me if the plugin should
 ExternalFilesPreferencePage_GenerateArchiveRepoAlways=&Expand the archive and generate the information necessary to install the archive contents
 ExternalFilesPreferencePage_GenerateArchiveRepoNever=Re&port the unrecognized archive content as an error
 ExternalFilesPreferencePage_GenerateArchiveRepoPrompt=As&k me if the information should be generated
-ExternalFilesPreferencePage_GenerateRepoAlways=&Generate the information necessary to install the directory contents
-ExternalFilesPreferencePage_GenerateRepoNever=&Report the unrecognized directory as an error
-ExternalFilesPreferencePage_GenerateRepoPrompt=&Ask me if the information should be generated
-ExternalFilesPreferencePage_RepoGenerationGroup=When I add an unrecognized directory as an update site
 UpdateAndInstallDialog_InstalledSoftware=&Installed Software
-UpdateAndInstallDialog_AccessibilityFilterDescription={0} {1} item showing
 UpdateAndInstallDialog_AddSiteButtonText=&Add Site...
 UpdateAndInstallDialog_AddSiteOperationlabel=Add Site
 UpdateAndInstallDialog_AvailableSoftware=&Available Software
 UpdateAndInstallDialog_AlertCheckbox=Automatically check for updates and alert me when they are found
 UpdateAndInstallDialog_PrefLink=Specify the update schedule and other <a>Update Preferences</a>
 UpdateAndInstallDialog_Title=Software Updates and Add-ons
-UpdateAndInstallDialog_TypeFilterText=Type filter text
 UpdateAndInstallDialog_ManageSites=&Manage sites...
 UpdateAndInstallDialog_RemoveSiteButtonText=&Remove Site...
 UpdateAndInstallDialog_RemoveSiteOperationLabel=Remove Site
@@ -48,13 +38,11 @@ UpdateAndInstallDialog_RevertActionLabel=&Revert configuration...
 UpdateAndInstallDialog_Properties=&Properties...
 UpdateAndInstallDialog_Refresh=&Refresh
 UpdateAndInstallDialog_ViewByCategory=&Category
-UpdateAndInstallDialog_ViewByMenuToolTip=View by
 UpdateAndInstallDialog_ViewByName=&Name
 UpdateAndInstallDialog_ViewBySite=&Site
 
 UpdateHandler_CannotLaunchUI=Cannot launch the Update UI.  This installation has not been configured properly for Software Updates.
 UpdateHandler_SDKUpdateUIMessageTitle=Software Updates
-RepositoryManipulationDialog_ErrorRetrievingSites=Error trying to retrieve the list of sites.
 RepositoryManipulationDialog_AddButton=&Add...
 RepositoryManipulationDialog_LocationColumnHeader=Location
 RepositoryManipulationDialog_NameColumnHeader=Name
@@ -94,7 +82,6 @@ SchedulerStartup_9PM=9:00 PM
 SchedulerStartup_10PM=10:00 PM
 SchedulerStartup_11PM=11:00 PM
 SchedulerStartup_12AM=12:00 AM
-AutomaticUpdatesJob_AutomaticUpdateSearch=Automatic Update Search
 AutomaticUpdatesPopup_PrefLinkOnly=Set up <a>Reminder options</a>
 AutomaticUpdatesPopup_RemindAndPrefLink=You will be reminded in {0}.\nSet reminder <a href="{1}">preferences</a>
 AutomaticUpdatesPopup_ReminderJobTitle=Update reminder job
@@ -103,26 +90,23 @@ AutomaticUpdateScheduler_30Minutes=30 minutes
 AutomaticUpdateScheduler_60Minutes=Hour
 AutomaticUpdateScheduler_240Minutes= 4 Hours
 AutomaticUpdateScheduler_UpdateNotInitialized=The update checker service is not initialized
-AutomaticUpdatesDialog_DownloadedNotification=Updates have been downloaded.  You may review and install them when you are ready.
-AutomaticUpdatesDialog_UpdatesAvailableTitle=Updates Available
+AutomaticUpdatesPopup_UpdatesAvailableTitle=Updates Available
 AutomaticUpdater_AutomaticDownloadOperationName=Automatic updates download
 AutomaticUpdater_ClickToReviewUpdates=Updates are available.  Click here to review and install them.
 AutomaticUpdater_ClickToReviewUpdatesWithProblems=Updates are available, but there may be some compatibility problems.  Click here to review them.
 AutomaticUpdater_ErrorCheckingUpdates=Unexpected error while checking the validity of updates.
-AutomaticUpdatesDialog_UpdatesFoundNotification=There are updates available for your software.  You may review and install them when you are ready.
 AutomaticUpdatesPreferencePage_UpdateSchedule=&Update schedule
 AutomaticUpdatesPreferencePage_findOnStart=Look for updates each time &platform is started
 AutomaticUpdatesPreferencePage_findOnSchedule=Look for updates on the following &schedule:
 AutomaticUpdatesPreferencePage_downloadOptions=&Download options
 AutomaticUpdatesPreferencePage_searchAndNotify=Search for updates and &notify me when they are available
 AutomaticUpdatesPreferencePage_downloadAndNotify=Download new updates &automatically and notify me when ready to install them
-AutomaticUpdatesDialog_ClickToReviewDownloaded=Software updates have been downloaded.\nClick to review and install updates.
-AutomaticUpdatesDialog_ClickToReviewNotDownloaded=Updates are available for your software.\nClick to review and install updates.
+AutomaticUpdatesPopup_ClickToReviewDownloaded=Software updates have been downloaded.\nClick to review and install updates.
+AutomaticUpdatesPopup_ClickToReviewNotDownloaded=Updates are available for your software.\nClick to review and install updates.
 AutomaticUpdatesPreferencePage_at=at
 AutomaticUpdatesPreferencePage_RemindGroup=&When updates are found
 AutomaticUpdatesPreferencePage_RemindSchedule=&Remind me about updates every:
 AutomaticUpdatesPreferencePage_RemindOnce=Notify me &once about updates
-AutomaticUpdatesJob_Updates=Updates
 ProvSDKQueryProvider_ErrorRetrievingProfile=Error while retrieving profile {0}
 ProvSDKUIActivator_ErrorWritingLicenseRegistry=Error writing license registry.  Accepted licenses will not be remembered.
 ProvSDKUIActivator_LaunchUpdateManager=Launch...
@@ -138,10 +122,6 @@ ProvisioningPreferencePage_AlwaysOpenWizard=&Open the wizard anyway to review or
 ProvisioningPreferencePage_BrowsingPrefsGroup=Browsing for updates
 ProvisioningPreferencePage_ShowLatestVersions=Show only the &latest versions of available software
 ProvisioningPreferencePage_ShowAllVersions=Show all &versions of available software
-ProvisioningPreferencePage_gcGroup=System cleanup
-ProvisioningPreferencePage_enableGC=Automatically delete unused files
-ProvisioningPreferencePage_gcImmediately=Delete unused files immediately
-ProvisioningPreferencePage_gcRetentionTime=Keep unused files for ???
 ProvisioningPreferencePage_NeverOpenWizard=&Report the problems and do not open the wizard
 ProvisioningPreferencePage_OpenWizardIfInvalid=When software selected for an install wizard may not be compatible
 ProvisioningPreferencePage_PromptToOpenWizard=&Ask me what to do when it happens
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/updates/AutomaticUpdatesPopup.java b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/updates/AutomaticUpdatesPopup.java
index 924f3aa..ddcad47 100644
--- a/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/updates/AutomaticUpdatesPopup.java
+++ b/bundles/org.eclipse.equinox.p2.ui.sdk/src/org/eclipse/equinox/internal/p2/ui/sdk/updates/AutomaticUpdatesPopup.java
@@ -56,7 +56,7 @@ public class AutomaticUpdatesPopup extends PopupDialog {
 	MouseListener clickListener;
 
 	public AutomaticUpdatesPopup(Shell parentShell, boolean alreadyDownloaded, Preferences prefs) {
-		super(parentShell, PopupDialog.INFOPOPUPRESIZE_SHELLSTYLE | SWT.MODELESS, false, true, false, false, ProvSDKMessages.AutomaticUpdatesDialog_UpdatesAvailableTitle, null);
+		super(parentShell, PopupDialog.INFOPOPUPRESIZE_SHELLSTYLE | SWT.MODELESS, false, true, false, false, ProvSDKMessages.AutomaticUpdatesPopup_UpdatesAvailableTitle, null);
 		downloaded = alreadyDownloaded;
 		this.prefs = prefs;
 		remindDelay = computeRemindDelay();
@@ -78,9 +78,9 @@ public class AutomaticUpdatesPopup extends PopupDialog {
 		// The "click to update" label
 		Label infoLabel = new Label(dialogArea, SWT.NONE);
 		if (downloaded)
-			infoLabel.setText(ProvSDKMessages.AutomaticUpdatesDialog_ClickToReviewDownloaded);
+			infoLabel.setText(ProvSDKMessages.AutomaticUpdatesPopup_ClickToReviewDownloaded);
 		else
-			infoLabel.setText(ProvSDKMessages.AutomaticUpdatesDialog_ClickToReviewNotDownloaded);
+			infoLabel.setText(ProvSDKMessages.AutomaticUpdatesPopup_ClickToReviewNotDownloaded);
 		infoLabel.setLayoutData(new GridData(GridData.FILL_BOTH));
 		infoLabel.addMouseListener(clickListener);
 
@@ -200,7 +200,7 @@ public class AutomaticUpdatesPopup extends PopupDialog {
 
 	protected void configureShell(Shell newShell) {
 		super.configureShell(newShell);
-		newShell.setText(ProvSDKMessages.AutomaticUpdatesDialog_UpdatesAvailableTitle);
+		newShell.setText(ProvSDKMessages.AutomaticUpdatesPopup_UpdatesAvailableTitle);
 	}
 
 	/**
commit b7de9932d4311f6957e249abac550383b126f087
Author: Susan Franklin <sfranklin>
Date:   Tue Apr 8 16:56:56 2008 +0000

    Bug 226116 - [ui] clean up unused messages

0	7	bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/ProvAdminUIMessages.java
0	6	bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/messages.properties
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/ProvAdminUIMessages.java b/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/ProvAdminUIMessages.java
index d7cdcc6..e8c1bad 100644
--- a/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/ProvAdminUIMessages.java
+++ b/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/ProvAdminUIMessages.java
@@ -27,13 +27,11 @@ public class ProvAdminUIMessages extends NLS {
 	public static String AddArtifactRepositoryDialog_DuplicateURL;
 	public static String AddArtifactRepositoryDialog_OperationLabel;
 	public static String AddProfileDialog_Title;
-	public static String AddRepositoryDialog_InvalidURL;
 	public static String AddMetadataRepositoryDialog_OperationLabel;
 	public static String MetadataRepositoriesView_AddRepositoryTooltip;
 	public static String MetadataRepositoriesView_AddRepositoryLabel;
 	public static String MetadataRepositoriesView_ChooseProfileDialogTitle;
 	public static String MetadataRepositoriesView_RemoveRepositoryTooltip;
-	public static String MetadataRepositoriesView_RemoveRepositoryOperationLabel;
 	public static String ArtifactRepositoriesView_AddRepositoryTooltip;
 	public static String ArtifactRepositoriesView_AddRepositoryLabel;
 	public static String ArtifactRepositoriesView_RemoveRepositoryTooltip;
@@ -55,16 +53,11 @@ public class ProvAdminUIMessages extends NLS {
 	public static String ProvisioningPrefPage_ShowGroupsOnly;
 	public static String ProvisioningPrefPage_ShowInstallRootsOnly;
 	public static String ProvisioningPrefPage_CollapseIUVersions;
-	public static String ProvisioningPrefPage_Description;
 	public static String ProvisioningPrefPage_UseCategories;
 
 	public static String Ops_RemoveProfileOperationLabel;
 	public static String AddProfileDialog_OperationLabel;
 	public static String AddProfileDialog_DuplicateProfileID;
-	public static String UpdateAndInstallDialog_AvailableIUsPageLabel;
-	public static String UpdateAndInstallDialog_InstalledIUsPageLabel;
-	public static String UpdateAndInstallDialog_Title;
-
 	public static String ProvView_RefreshCommandLabel;
 	public static String ProvView_RefreshCommandTooltip;
 
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/messages.properties b/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/messages.properties
index 07184ab..82a168f 100644
--- a/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.ui.admin/src/org/eclipse/equinox/internal/p2/ui/admin/messages.properties
@@ -14,14 +14,12 @@ ProvAdminUIActivator_ContinueAnyway=Continue
 ProvAdminUIActivator_LaunchUpdateManager=Launch UM...
 ProvAdminUIActivator_UnsupportedInstallHandler=Unsupported Install Handler
 ProvAdminUIActivator_UnsupportedInstallHandlerMessage=The feature you are trying to install has defined an Update Manager install handler.  You may launch the old Update Manager in order to get the install handler support, or continue the install and ignore the install handler.
-ProvisioningPrefPage_Description=Provisioning
 ProvisioningPrefPage_ShowGroupsOnly=Show only &group IU's in profile or repository views
 ProvisioningPrefPage_ShowInstallRootsOnly=Show only the install &roots in a profile
 ProvisioningPrefPage_HideSystemRepos=Hide &system repositories in repository views
 ProvisioningPrefPage_CollapseIUVersions=Collapse multiple IU &versions underneath latest version
 ProvisioningPrefPage_UseCategories=Show IUs by &category in repository views
 InstalledIUPropertyPage_NoInfoAvailable=No profile-specific properties are available.
-AddRepositoryDialog_InvalidURL=Invalid URL
 AddProfileDialog_Title=Add Profile
 AddProfileDialog_OperationLabel=Add Profile
 AddMetadataRepositoryDialog_OperationLabel=Add Metadata Repository
@@ -36,7 +34,6 @@ MetadataRepositoriesView_RemoveRepositoryOperationLabel=Remove Metadata Reposito
 ArtifactRepositoriesView_AddRepositoryLabel=Add repository...
 ArtifactRepositoriesView_AddRepositoryTooltip=Add a new artifact repository
 ArtifactRepositoriesView_RemoveRepositoryTooltip=Remove the selected artifact repositories
-ArtifactRepositoriesView_RemoveRepositoryOperationLabel=Remove Artifact Repositories
 ProfilesView_ErrorRetrievingProfiles=Error retrieving profiles
 ProfilesView_AddProfileTooltip=Add a new profile
 ProvView_RefreshCommandTooltip=Refresh the items in the view.
@@ -46,9 +43,6 @@ ProfilesView_RemoveProfileLabel=Remove
 ProfilesView_RemoveProfileTooltip=Remove the selected profiles
 AddProfileDialog_DuplicateProfileID=The profile ID is already in use.
 Ops_RemoveProfileOperationLabel=Remove Profiles
-UpdateAndInstallDialog_InstalledIUsPageLabel=Installed IUs
-UpdateAndInstallDialog_AvailableIUsPageLabel=Available IUs
-UpdateAndInstallDialog_Title=Install/Update
 ProfileRootPropertyName=Profile Root
 RepositoryGroup_PropertiesLabel=Repository properties:
 RepositoryGroup_NameColumnLabel=Name
commit 683072167e6f86ea576eb89b10890c77ac99ffe7
Author: Susan Franklin <sfranklin>
Date:   Tue Apr 8 17:11:55 2008 +0000

    Bug 226116 - [ui] clean up unused messages

0	29	bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java
0	29	bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties
diff --git a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java
index 37d0939..d1392ce 100644
--- a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java
+++ b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIMessages.java
@@ -36,7 +36,6 @@ public class ProvUIMessages extends NLS {
 	public static String AcceptLicensesWizardPage_Title;
 	public static String ApplyProfileChangesDialog_ApplyChanges;
 	public static String CategoryElementCollector_Uncategorized;
-	public static String ChooseProfileDialog_DefaultDialogTitle;
 	public static String InstallDialog_InstallSelectionMessage;
 	public static String IUCopyrightPropertyPage_NoCopyright;
 	public static String IUCopyrightPropertyPage_ViewLinkLabel;
@@ -55,35 +54,22 @@ public class ProvUIMessages extends NLS {
 	public static String IULicensePropertyPage_NoLicense;
 	public static String IULicensePropertyPage_ViewLicenseLabel;
 	public static String ProfileElement_InvalidProfile;
-	public static String ProfileModificationAction_NullPlan;
 	public static String ProfileModificationAction_UnexpectedError;
 	public static String ProfileModificationWizardPage_DetailsLabel;
 	public static String ProfileModificationWizardPage_NothingSelected;
-	public static String ProfileModificationWizardPage_ProfileNotFound;
 	public static String ProfileModificationWizardPage_UnexpectedError;
 	// utility error messages
 	public static String ProvisioningUtil_NoRepositoryManager;
-	public static String ProvisioningUtil_AddRepositoryFailure;
-	public static String ProvisioningUtil_CreateRepositoryFailure;
-	public static String ProvisioningUtil_InstallManyTask;
-	public static String ProvisioningUtil_InstallOneTask;
 	public static String ProvisioningUtil_LoadRepositoryFailure;
-	public static String ProvisioningUtil_RepoNotWritable;
-	public static String ProvisioningUtil_RepositoryNotFound;
 	public static String ProvisioningUtil_NoProfileRegistryFound;
 	public static String ProvisioningUtil_NoPlannerFound;
 	public static String ProvisioningUtil_NoDirectorFound;
 	public static String ProvisioningUtil_NoEngineFound;
-	public static String ProvisioningUtil_NoIUFound;
-	public static String ProvisioningUtil_NoInstallRegistryFound;
-	public static String ProvisioningUtil_NoProfileInstallRegistryFound;
-	public static String ProvisioningUtil_RepositoriesSearched;
 
 	// viewer support
 	public static String ProvDropAdapter_InvalidDropTarget;
 	public static String ProvDropAdapter_NoIUsToDrop;
 	public static String ProvDropAdapter_UnsupportedDropOperation;
-	public static String ProvElementQueryResult_CouldNotInstantiateElement;
 
 	// Provisioning operations
 	public static String ProvisioningOperation_ExecuteErrorTitle;
@@ -91,23 +77,19 @@ public class ProvUIMessages extends NLS {
 	public static String ProvisioningOperation_UndoErrorTitle;
 	public static String ProvisioningOperationRunner_ErrorExecutingOperation;
 	public static String InstallIUOperationLabel;
-	public static String InstallIUOperationLabelWithMnemonic;
 	public static String InstallIUCommandLabel;
 	public static String InstallIUCommandTooltip;
 	public static String InstallIUProgress;
 	public static String UninstallDialog_UninstallMessage;
 	public static String UninstallIUOperationLabel;
-	public static String UninstallIUOperationLabelWithMnemonic;
 	public static String UninstallIUCommandLabel;
 	public static String UninstallIUCommandTooltip;
 	public static String UninstallIUProgress;
 	public static String UpdateIUOperationLabel;
-	public static String UpdateIUOperationLabelWithMnemonic;
 	public static String UpdateIUCommandLabel;
 	public static String UpdateIUCommandTooltip;
 	public static String UpdateIUProgress;
 	public static String RevertIUOperationLabel;
-	public static String RevertIUOperationLabelWithMnemonic;
 	public static String RevertIUCommandLabel;
 	public static String RevertIUCommandTooltip;
 	public static String RevertIUProgress;
@@ -133,19 +115,12 @@ public class ProvUIMessages extends NLS {
 	public static String AddRepositoryDialog_DuplicateURL;
 	public static String AddRepositoryDialog_InvalidURL;
 	public static String AddRepositoryDialog_Title;
-	public static String AddRepositoryDialog_URLValidationError;
-	public static String AvailableIUContentProvider_FailureRetrievingContents;
-	public static String AvailableIUContentProvider_JobName;
-	public static String AvailableIUContentProvider_PlaceholderLabel;
 	public static String AvailableIUElement_ProfileNotFound;
 	public static String AvailableIUGroup_ViewByToolTipText;
 	public static String Label_Profiles;
 	public static String Label_Repositories;
 	public static String MetadataRepositoryElement_RepositoryLoadError;
-	public static String UpdateAction_ExceptionDuringUpdateCheck;
-	public static String UpdateAction_UpdateInformationTitle;
 	public static String UpdateAction_UpdatesAvailableMessage;
-	public static String UpdateDialog_AssemblingUpdatesProgress;
 	public static String UpdateAction_UpdatesAvailableTitle;
 	public static String PlatformUpdateTitle;
 	public static String PlatformRestartMessage;
@@ -154,7 +129,6 @@ public class ProvUIMessages extends NLS {
 	public static String ProvUI_NameColumnTitle;
 	public static String ProvUI_SizeColumnTitle;
 	public static String ProvUI_VersionColumnTitle;
-	public static String ProvUI_IDColumnTitle;
 	public static String ProvUI_WarningTitle;
 	public static String ProvUIActivator_ExceptionDuringProfileChange;
 	public static String ProvUILicenseManager_ParsingError;
@@ -171,13 +145,10 @@ public class ProvUIMessages extends NLS {
 	public static String RevertDialog_PageTitle;
 	public static String RevertDialog_RevertError;
 	public static String RevertDialog_RevertOperationLabel;
-	public static String RevertDialog_SelectMessage;
 	public static String RevertDialog_Title;
 	public static String RevertProfileWizardPage_ErrorRetrievingHistory;
 
 	// Operations
-	public static String UpdateAndInstallSelectionDialog_DeselectAllLabel;
-	public static String UpdateAndInstallSelectionDialog_SelectAllLabel;
 	public static String URLValidator_UnrecognizedURL;
 	public static String UpdateManagerCompatibility_UnableToOpenFindAndInstall;
 	public static String UpdateManagerCompatibility_UnableToOpenManageConfiguration;
diff --git a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties
index 0f5d358..b73ed70 100644
--- a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties
+++ b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/messages.properties
@@ -10,58 +10,41 @@
 ###############################################################################
 
 ProfileElement_InvalidProfile=Invalid profile ID {0}
-ProfileModificationAction_NullPlan=Unexpected error while resolving.  Null provisioning plan encountered.
 ProfileModificationAction_UnexpectedError=Unexpected error encountered while preparing for the operation.
 ProfileModificationWizardPage_DetailsLabel=Details
 ProfileModificationWizardPage_NothingSelected=You must check some items in order to continue.
-ProfileModificationWizardPage_ProfileNotFound=Could not resolve profile ID.
 ProfileModificationWizardPage_UnexpectedError=An unexpected error occurred.  This wizard is no longer valid with the current selections.
 ProvisioningUtil_NoRepositoryManager=No repository manager was found
-ProvisioningUtil_AddRepositoryFailure=Unable to add repository {0}
-ProvisioningUtil_CreateRepositoryFailure=Unable to create repository {0}
-ProvisioningUtil_InstallManyTask=Installing {0} items into {1}
-ProvisioningUtil_InstallOneTask=Installing {0} into {1}
 ProvisioningUtil_LoadRepositoryFailure=Unable to load the repository {0}
-ProvisioningUtil_RepositoryNotFound=Could not find repository {0}
-ProvisioningUtil_RepoNotWritable=Repository is not writable
 ProvisioningUtil_NoProfileRegistryFound=No profile registry was found
 ProvisioningUtil_NoPlannerFound=No provisioning planner was found
 ProvisioningUtil_NoDirectorFound=No provisioning director was found
 ProvisioningUtil_NoEngineFound=No provisioning engine was found
-ProvisioningUtil_NoIUFound=Could not find installable unit {0}:{1}
-ProvisioningUtil_NoInstallRegistryFound=No install registry was found
-ProvisioningUtil_NoProfileInstallRegistryFound=No profile install registry was found
-ProvisioningUtil_RepositoriesSearched=The following repositories were searched:
 
 ProvisioningOperation_ExecuteErrorTitle=Error while executing the "{0}" operation
 ProvisioningOperation_RedoErrorTitle=Error while redoing the "{0}" operation
 ProvisioningOperation_UndoErrorTitle=Error while undoing the "{0}" operation
 ProvisioningOperationRunner_ErrorExecutingOperation=Unexpected error while trying to run {0}
 InstallIUOperationLabel=Install
-InstallIUOperationLabelWithMnemonic=&Install
 InstallIUCommandLabel=&Install...
 InstallIUCommandTooltip=Install the selected items
 InstallIUProgress=Installing...
 UninstallDialog_UninstallMessage=Review and confirm that the checked items will be uninstalled.
 UninstallIUOperationLabel=Uninstall
-UninstallIUOperationLabelWithMnemonic=&Uninstall
 UninstallIUCommandLabel=&Uninstall...
 UninstallIUCommandTooltip=Uninstall the selected items
 UninstallIUProgress=Uninstalling...
 UpdateIUOperationLabel=Update
-UpdateIUOperationLabelWithMnemonic=&Update
 UpdateIUCommandLabel=&Check for updates...
 UpdateIUCommandTooltip=See if updates are available for the selected items
 UpdateIUProgress=Updating...
 RevertIUOperationLabel=Revert
-RevertIUOperationLabelWithMnemonic=&Revert
 RevertIUCommandLabel=&Revert...
 RevertIUCommandTooltip=Revert to a previously installed configuration
 RevertIUProgress=Reverting...
 ProvDropAdapter_NoIUsToDrop=No IUs were available to be dropped
 ProvDropAdapter_InvalidDropTarget=Invalid drop target
 ProvDropAdapter_UnsupportedDropOperation=Unsupported drop operation
-ProvElementQueryResult_CouldNotInstantiateElement=Could not instantiate UI element class {0}
 
 IUCopyrightPropertyPage_NoCopyright=No copyright information has been provided.
 IUCopyrightPropertyPage_ViewLinkLabel=View full copyright:
@@ -106,21 +89,14 @@ AcceptLicensesWizardPage_Title=Review Licenses
 AddRepositoryDialog_Title=Add Repository
 AddRepositoryDialog_DuplicateURL=Duplicate URL
 AddRepositoryDialog_InvalidURL=The provided URL is not valid
-AddRepositoryDialog_URLValidationError=Unexpected failure while validating the location.
 
-UpdateAndInstallSelectionDialog_DeselectAllLabel=&Deselect All
-UpdateAndInstallSelectionDialog_SelectAllLabel=&Select All
-UpdateAction_ExceptionDuringUpdateCheck=Error encountered while checking for updates.
-UpdateAction_UpdateInformationTitle=Update Information
 UpdateAction_UpdatesAvailableMessage=Review and confirm that the checked updates will be installed.
-UpdateDialog_AssemblingUpdatesProgress=Assembling updates
 UpdateAction_UpdatesAvailableTitle=Available Updates
 UpdateOperation_NothingToUpdate=There is nothing to update.
 URLValidator_UnrecognizedURL=The URL {0} is not recognized.
 UpdateManagerCompatibility_UnableToOpenFindAndInstall=Unable to open the Update Manager Find and Install Wizard.
 UpdateManagerCompatibility_UnableToOpenManageConfiguration=Unable to open the Update Manager Configuration Manager.
 CategoryElementCollector_Uncategorized=Other
-ChooseProfileDialog_DefaultDialogTitle=Select a profile
 InstallDialog_InstallSelectionMessage=Review and confirm that the checked items will be installed.
 
 PlatformUpdateTitle = Software Updates
@@ -131,7 +107,6 @@ ProvUI_InformationTitle=Information
 ProvUI_NameColumnTitle=Name
 ProvUI_SizeColumnTitle=Size
 ProvUI_VersionColumnTitle=Version
-ProvUI_IDColumnTitle=Identifier
 ProvUI_WarningTitle=Warning
 ProvUIActivator_ExceptionDuringProfileChange=Exception while handling profile change event.
 ProvUILicenseManager_ParsingError=Error parsing the accepted license registry
@@ -140,9 +115,6 @@ OptionalPlatformRestartMessage = It is recommended you restart the system for th
 but it may be possible to apply the changes you have made without restarting. \
 Would you like to restart now?
 ApplyProfileChangesDialog_ApplyChanges=Apply Changes
-AvailableIUContentProvider_FailureRetrievingContents={0} is not available.
-AvailableIUContentProvider_JobName=Fetching contents for {0}
-AvailableIUContentProvider_PlaceholderLabel=Retrieving contents of {0}...
 AvailableIUElement_ProfileNotFound=Could not find profile to use for size computation.
 AvailableIUGroup_ViewByToolTipText=View by
 Label_Profiles=All Software Profiles
@@ -161,7 +133,6 @@ RevertDialog_Description=Choose a previously installed configuration to which yo
 RevertDialog_PageTitle=Revert to a Previous Configuration
 RevertDialog_RevertError=Unexpected error while reverting configuration.  The configuration was not reverted.
 RevertDialog_RevertOperationLabel=Revert configuration
-RevertDialog_SelectMessage=Select a configuration from the list on the left.  The contents of that configuration will be shown on the right.
 RevertDialog_Title=Revert Software Configuration
 
 RevertProfileWizardPage_ErrorRetrievingHistory=Unexpected error while getting the history list.
